backend/test/db.ts

```diff
@@ -0,0 +1,15 @@
+import { TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { EntitySchema } from 'typeorm';
+
+type Entity = Function | string | EntitySchema<any>;
+
+export const createTestConfiguration = (
+  entities: Entity[],
+): TypeOrmModuleOptions => ({
+  type: 'sqlite',
```

Comment:
If we agree to remove the sqlite, then perhaps this can be changed to `postgres`